axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,158 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+class Tranquilizer(Player):
+   
+    """
+    Submitted to Axelrod's second tournament by Craig Feathers
+
+    This strategy is based on the reverse engineering of the 
+    Fortran strategy K67R from Axelrod's second tournament.
+    Reversed engineered by: Owen Campbell, Will Guo and Mansour Hakem.
+
+
+    Description given in Axelrod's "More Effective Choice in the Prisoner's Dilemma" 
+    paper: The rule normally cooperates but is ready to defect if the other player 
+    defects too often. Thus the rule tends to cooperate for the first dozen 
+    or two moves if the other player is cooperating, but then it throws in a 
+    defection. If the other player continues to cooperate, then defections 
+    become more frequent. But as long as Tranquilizer is maintaining an 
+    average payoff of at least 2.25 points per move, it will never defect 
+    twice in succession and it will not defect more than one-quarter of the time.
+
+    
+    Has a variable, 'FD' which can be 0, 1 or 2. It has an initial value of 0
+    Has a variable 'S', which counts the consecutive number of times the opponent has played D (i.e. it is reset to 0 if the opponent plays C). It has an initial value of 0.
+    Has a variable, 'C', which counts the number of times the opponent Cooperates
+    Has a variable 'AK' which increases each time a move is played whilst in state FD = 1. It has an initial value of 1.
+    Has a variable 'NK' which increases each time a move is played whilst in state FD = 2. It has an initial value of 1.
+    Has a variable 'AD' with an initial value of 5
+    Has a variable 'NO with an initial value of 0
+    When FD = 0:
```

Comment:
Currently your docs are failing on the automated testing: https://travis-ci.org/Axelrod-Python/Axelrod/jobs/267887570

This is because of how [sphinx](http://www.sphinx-doc.org/en/stable/rest.html) formats things. The following will fix it:


```

 232     Has a variable 'NO with an initial value of 0                                                                                                                                                                                                                       
~ 233                                                                                                                                                                                                                                                                         
~ 234                                                                                                                                                                                                                                                                         
~ 235     The strategy follows the following algorithm::                                                                                                                                                                                                                      
~ 236                                                                                                                                                                                                                                                                         
~ 237         When FD = 0:                                                                                                                                                                                                                                                    
~ 238                                                                                                                                                                                                                                                                         
~ 239             If the opponent's last move (JA) was Cooperate, increase the value of C by 1                                                                                                                                                                                
~ 240             If Score (K) < 1.75 * Move Number (M), play opponent's last move                                                                                                                                                                                            
~ 241             If (1.75 * M) <= K < (2.25 * M):                                                                                                                                                                                                                            
~ 242                                                                                                                                                                                                                                                                         
~ 243                 Calculate Probability P:                                                                                                                                                                                                                                
~ 244                 P = 0.25 + C/M - 0.25*S + (K - L)/100 + 4/M                                                                                                                                                                                                             
~ 245                 Where L is the opponent's score so far                                                                                                                                                                                                                  
~ 246                 If Random (R) <= P:                                                                                                                                                                                                                                     
~ 247                                                                                                                                                                                                                                                                         
~ 248                     Cooperate                                                                                                                                                                                                                                           
~ 249                 Else:                                                                                                                                                                                                                                                   
~ 250                                                                                                                                                                                                                                                                         
~ 251                     Defect                                                                                                                                                                                                                                              
~ 252                                                                                                                                                                                                                                                                         
~ 253             If K >= (2.25 * M):                                                                                                                                                                                                                                         
~ 254                                                                                                                                                                                                                                                                         
~ 255                 Calculate probability P:                                                                                                                                                                                                                                
~ 256                 P = 0.95 - (AD + NO - 5)/15 + 1/M**2 - J/4                                                                                                                                                                                                              
~ 257                 Where J is the opponent's last move                                                                                                                                                                                                                     
~ 258                                                                                                                                                                                                                                                                         
~ 259                 If Random (R) <= P:                                                                                                                                                                                                                                     
~ 260                                                                                                                                                                                                                                                                         
~ 261                     Cooperate                                                                                                                                                                                                                                           
~ 262                                                                                                                                                                                                                                                                         
~ 263                 Else:                                                                                                                                                                                                                                                   
~ 264                                                                                                                                                                                                                                                                         
~ 265                     Set FD = 1                                                                                                                                                                                                                                          
~ 266                     Defect                                                                                                                                                                                                                                              
~ 267                                                                                                                                                                                                                                                                         
+ 268         When FD = 1:                                                                                                                                                                                                                                                    
+ 269                                                                                                                                                                                                                                                                         
+ 270             Set FD = 2                                                                                                                                                                                                                                                  
+ 271             Set the variable 'AD':                                                                                                                                                                                                                                      
+ 272             AD = ((AD * AK) + 3 - (3 * J) + (2 * JA) - (JA * J)) / (AK + 1)                                                                                                                                                                                             
+ 273             Where JA is the strategy's last move and J is the opponent's last move (C = 0, D = 1)                                                                                                                                                                       
+ 274             Increase the value of AK by 1                                                                                                                                                                                                                               
+ 275             Cooperate                                                                                                                                                                                                                                                   
+ 276                                                                                                                                                                                                                                                                         
+ 277         When FD = 2:                                                                                                                                                                                                                                                    
+ 278                                                                                                                                                                                                                                                                         
+ 279             Set FD = 0                                                                                                                                                                                                                                                  
+ 280             Set the variable 'NO':                                                                                                                                                                                                                                      
+ 281             NO = ((NO * NK) + 3 - (3 * J) + (2 * JA) - (JA * J) / (NK + 1)                                                                                                                                                                                              
+ 282             Where JA the strategy's last move and J is the opponent's last move (C = 0, D = 1)                                                                                                                                                                          
+ 283             Increase the value of NK by 1                                                                                                                                                                                                                               
+ 284             Cooperate                                                                                                                                                                                                                                                   
+ 285                                                                                                                                                                                                                                                                         
  286     Tranquilizer came in 27th place in Axelrod's second torunament. 

```